### PR TITLE
WW-493: Add text about creating wiki on last CNW step

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
@Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 1]
```diff
@@ -606,6 +606,7 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 				},
 				callback: function (res) {
 					throbberWrapper.stopThrobbing();
+					throbberWrapper.removeClass('creating-wiki');
 					cityId = res.cityId;
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:610]: MAJOR Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 2]
```diff
@@ -606,6 +606,7 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
...
 					createStatus = res.status;
 					createStatusMessage = res.statusMsg;
```
Comment (WikiaTech) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:612]: MAJOR Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* MAJOR 5 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/76d1b361c8b3872482a9b6f430ef047871089ea8/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/76d1b361c8b3872482a9b6f430ef047871089ea8/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L475](https://github.com/Wikia/app/blob/76d1b361c8b3872482a9b6f430ef047871089ea8/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L475): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 3]
```diff
@@ -306,6 +306,25 @@
...
+#ThemeWiki {
+	.creating-wiki-message {
+		display: none;
+	}
+
+	.creating-wiki {
+		.creating-wiki-message {
+			color: $wds-color-light-gray;
+			display: block;
+			font-size: 14px;
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: magic number

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 4]
```diff
@@ -306,6 +306,25 @@
...
+			padding: 16px 18px 0 0;
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: magic numbers. aren't these paddings defined somewhere?

File: extensions/wikia/CreateNewWiki/js/WikiBuilder.js [hunk 5]
```diff
@@ -606,6 +606,7 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 				},
 				callback: function (res) {
 					throbberWrapper.stopThrobbing();
```
Comment (rybmat) [inline extensions/wikia/CreateNewWiki/js/WikiBuilder.js:608]: I don't remember what we decided about throbber itself, are we leaving the old one or should we change it to spinner?

**Review by cormorando — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* MAJOR 5 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/16aec5bf3fc15a04523406250d9d4a28bb7f641b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/16aec5bf3fc15a04523406250d9d4a28bb7f641b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L475](https://github.com/Wikia/app/blob/16aec5bf3fc15a04523406250d9d4a28bb7f641b/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L475): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Review by rybmat — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
- MAJOR 5 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/add9a67b594caee4207e4ba1842b5279c297a043/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/add9a67b594caee4207e4ba1842b5279c297a043/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L475](https://github.com/Wikia/app/blob/add9a67b594caee4207e4ba1842b5279c297a043/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L475): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)

**Pull request merged**